web/frontend/src/app/component/concept-display/concept-display.component.ts

```diff
@@ -5,7 +5,7 @@ import { ConceptDetailService } from './../../service/concept-detail.service';
 import { Concept } from './../../model/concept';
 import { ConfigurationService } from '../../service/configuration.service';
 import { Subject } from 'rxjs';
-import { writeXLSX, utils, WorkSheet } from 'xlsx';
+import { writeXLSX, utils, WorkSheet } from 'xlsx-republish';
```

Comment:
I tried running the frontend locally, npm start, and I'm getting an error with this import causing the system to fail compile
```
./src/app/component/concept-display/concept-display.component.ts:12:0-50 - Error: Module not found: Error: Can't resolve 'xlsx-republish' in '/Users/gaalicia/Documents/nci/evs-explore/web/frontend/src/app/component/concept-display'

Error: src/app/component/concept-display/concept-display.component.ts:8:45 - error TS2307: Cannot find module 'xlsx-republish' or its corresponding type declarations.

8 import { writeXLSX, utils, WorkSheet } from 'xlsx-republish';
 ```

Reply:
`npm i` or `npm install xlsx-republish` should fix that